Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -269,6 +267,29 @@ def collect_tests_and_content_packs(
     return test_ids, ids_with_no_tests, caught_missing_test, packs_to_install
 
 
+def check_if_test_should_not_be_missed(test_playbook_path: str, test_playbook_name: str) -> bool:
+    """
+    Checks if a missing test from conf JSON should be missed or not.
+    Test should be missed if the support level is not XSOAR.
+    Args:
+        test_playbook_path (str): Path to test playbook.
+        test_playbook_name (str): Name of the test playbook. For logging purposes.
+
+    Returns:
+        (bool): Whether test should be missed.
+    """
+    pack_name: Optional[str] = tools.get_pack_name(test_playbook_path)
+    if tools.get_pack_metadata(test_playbook_path).get(PACK_METADATA_SUPPORT, '') == 'xsoar':
```

Comment:
Can we edit `is_pack_xsoar_supported` to use the `tools.get_pack_metadata`like here and then use it? I want that there will be one place in the code that checks if a pack is xsoar supported, not several,

Reply:
yep, fixing